api/nakadi-event-bus-api.yaml

```diff
@@ -1567,13 +1651,14 @@ definitions:
           future there could be others.
       schema:
         type: string
-        $ref: '#/definitions/EventTypeSchema'
         description: |
           The schema as string in the syntax defined in the field type. Failure to respect the
           syntax will fail any operation on an EventType.
 
-          To have a generic, undefined schema it is possible to define the schema as `"schema":
-          "{\"additionalProperties\": true}"`.
+          When defining a schema, every property should be explicitly defined with its type. It means that,
+          additionalProperties true is not accepted.
```

Comment:
@rcillo That sounds good (and sensible - if starting as "extensible" then it makes perfect sense that that cannot then later be constrained (as you say, constraint may not hold for older data).

I'm not sure I follow your point on `additionalProperties` though - is it that we will it be possible to use `additionalProperties` to define a map of some specific type, but, that map's definition cannot change after that? Or something else?